lib/svg-sprite/calculate-svg-dimensions.js

```diff
@@ -1,32 +1,31 @@
 'use strict';
-
-/* eslint-env browser */
+const resvg = require('@resvg/resvg-js');
+const imageSize = require('image-size');
 
 /**
  * Calculate an SVG rendered dimensions.
- * It truncates everything before the first '<svg ' string
  *
  * @param svg {string}
- * @param browser {puppeteer.Browser}
- * @returns {Promise<{ height:number, width: number }>}
+ * @returns {Promise<{ height: Number, width: Number }>}
  */
-async function calculateSvgDimensions(svg, browser) {
-    let page;
-
+async function calculateSvgDimensions(svg) {
     try {
-        page = await browser.newPage();
-        const html = `<svg xmlns="http://www.w3.org/2000/svg">${svg.substr(svg.toLowerCase().indexOf('<svg'))}</svg>`;
-        await page.setContent(html, { waitUntil: 'networkidle0' });
+        const pngData = await resvg.renderAsync(svg, {
+            fitTo: { mode: 'original' },
+            logLevel: 'error'
+        });
 
-        return await page.evaluate(() => {
-            const { height, width } = document.getElementsByTagName('svg')[1].getBoundingClientRect();
+        const dimensions = imageSize.imageSize(pngData);
```

Comment:
Maybe I can return the width and height of the png file directly in [resvg-js](https://github.com/yisibl/resvg-js). I need your feedback.

Reply:
@yisibl what'll be great. But I think this should be a new method.

Reply:
Okay.